src/common/main.lua

```diff
@@ -16,10 +16,20 @@ main.init = function()
 	-- ADDRESSES["userAddress"] = {"antProcessId1" = true, "antProcessId2" = true}
 	ADDRESSES = ADDRESSES or {}
 
+	ANTVersions = ANTVersions
+		or {
+			["1.0.0"] = {
```

Comment:
what's the motivation behind semantic versioning here? a breaking change would suggest breaking the spec. how do you intend to manage versions of ANTs?

Reply:
Will change - stepping away from semver to iterative versions